src/command_modules/azure-cli-eventhubs/azure/cli/command_modules/eventhubs/_params.py

```diff
@@ -76,7 +76,7 @@ def load_arguments_eh(self, _):
     for scope in ['eventhubs eventhub update', 'eventhubs eventhub create']:
         with self.argument_context(scope) as c:
             c.argument('message_retention_in_days', options_list=['--message-retention'], type=int, help='Number of days to retain events for this Event Hub, value should be 1 to 7 days')
-            c.argument('partition_count', type=int, help='Number of partitions created for the Event Hub, allowed values are from 1 to 32 partitions.')
+            c.argument('partition_count', type=int, help='Number of partitions created for the Event Hub, allowed values are from 2 to 32 partitions. Default lower limit of 1 is supported with Kafka enabled namespaces. Default upper limit is 32 unless you have custom quota, in which case it would be upper limit of that quota.')
```

Comment:
It sounds from the additional help you added that the allowed values are not restrained from 2-32? At least for some users?

Reply:
yes, by default the upper limit is of 32, which can be increased by raising the quota request.

Reply:
Right, my point is, the wording: "allowed values are from 2 to 32 partitions" implies a strict boundary, i.e. NO ONE is able to specify `--partition-count 1` or `--partition-count 33`.

The wording could be improved. Recommend:
`By default, allowed values are 2-32. Value of 1 is supported with Kafka enabled namespaces. In presence of a custom quota, the upper limit will match the upper limit of the quota.`

Reply:
updated with : Number of partitions created for the Event Hub. By default, allowed values are 2-32. Lower value of 1 is supported with Kafka enabled namespaces. In presence of a custom quota, the upper limit will match the upper limit of the quota.